pkg/api/bootstrap.go

```diff
@@ -62,10 +62,17 @@ func (*RealUrlResolutionService) Get(url string) (string, error) {
 		// Make sure the http response is closed (eventually).
 		defer resp.Body.Close()
```

Comment:
Should we protect against resp being null ? or resp.Body being null ?

Reply:
Go's http.Get call works differently to the OpenAPI-generated client code in that it always returns a response body if no error came back from the call. See the [http.Get docs](https://pkg.go.dev/net/http#Get):

```
When err is nil, resp always contains a non-nil resp.Body. Caller should close resp.Body when done reading from it.
```